src/components/signUp/Details.js

```diff
@@ -0,0 +1,200 @@
+import React, { useEffect, useState } from 'react'
+import { useForm } from "react-hook-form";
+import { useNavigate } from "react-router-dom";
+import { useDispatch, useSelector } from "react-redux";
+import { Link } from "react-router-dom";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { UserDetailSchema } from "../../schema/signUpSchema";
+import Input from '../input';
+import Button from '../button';
+import { setCurrentStepper,setData } from "../../redux/features/slices/stepper";
+const Details = () => {
```

Comment:
What's the meaning of const Deatails ... in signup?

Reply:
UserDetails